File: TransCoder_tokenized_test_set_functions/java/SQUARES_OF_MATRIX_DIAGONAL_ELEMENTS_1.java
static void diagonalsquare ( int mat [ ] [ ], int row, int column ) {
  System . out . print ( " Diagonal one : " );
  for ( int i = 0;
  i < row;
  i ++ ) {
    System . out . print ( mat [ i ] [ i ] * mat [ i ] [ i ] + " " );
  }
  System . out . println ( );
  System . out . print ( " Diagonal two : " );
  for ( int i = 0;
  i < row;
  i ++ ) {
    System . out . print ( mat [ i ] [ row - i - 1 ] * mat [ i ] [ row - i - 1 ] + " " );
  }
}

File: TransCoder_tokenized_test_set_functions/python/SQUARES_OF_MATRIX_DIAGONAL_ELEMENTS_1.py
def diagonalsquare ( mat, row, column ) :
    print ( "Diagonal one : ", end = "" )
    for i in range ( 0, row ) :
        print ( mat [ i ] [ i ] * mat [ i ] [ i ], end = " " )
    print ( "\n\nDiagonal two : ", end = "" )
    for i in range ( 0, row ) :
        print ( mat [ i ] [ row - i - 1 ] * mat [ i ] [ row - i - 1 ], end = " " )
    
